addons/Lookup/Lookup.lua

```diff
@@ -0,0 +1,190 @@
+--[[
+	Copyright (c) 2018, Karuberu
+	All rights reserved.
+
+	Redistribution and use in source and binary forms, with or without
+	modification, are permitted provided that the following conditions are met:
+
+	    * Redistributions of source code must retain the above copyright
+	      notice, this list of conditions and the following disclaimer.
+	    * Redistributions in binary form must reproduce the above copyright
+	      notice, this list of conditions and the following disclaimer in the
+	      documentation and/or other materials provided with the distribution.
+	    * Neither the name of Lookup nor the
+	      names of its contributors may be used to endorse or promote products
+	      derived from this software without specific prior written permission.
+
+	THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+	ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+	WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+	DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDERS OR CONTRIBUTORS BE LIABLE FOR ANY
+	DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+	(INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+	LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+	ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+	(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+	SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+]]
+_addon.name = 'Lookup'
+_addon.author = 'Karuberu'
+_addon.version = '1.0'
+_addon.language = 'english'
```

Comment:
This line is bad, and you're not at fault for writing this, because many people do (including the tool who wrote Linker). This effectively forces lookup to be made with English names throughout your addon, which would suck for Japanese players.

Reply:
Oh, interesting... So I should just remove that `_addon.language = 'english'` line, then?

It may be helpful to make some note about that technicality to the [Information section of the Writing Addons wiki page](https://github.com/Windower/Lua/wiki/Writing-Addons#information), which seems to recommend adding it.

Reply:
Yes, please, I also removed it from that guide, thanks for pointing that out. I implemented the `_addon.language` field with some grand ideas in mind that turned out to be drunken misconceptions.